<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Control Key</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Config</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="20" column="2">
         <widget class="QComboBox" name="comboBox_start"/>
        </item>
        <item row="23" column="1">
         <widget class="QLabel" name="label_32">
          <property name="text">
           <string>Guide</string>
          </property>
         </widget>
        </item>
        <item row="17" column="1">
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>DPad Left</string>
          </property>
         </widget>
        </item>
        <item row="9" column="1">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Right Bumper</string>
          </property>
         </widget>
        </item>
        <item row="8" column="2">
         <widget class="QComboBox" name="comboBox_lshoulder"/>
        </item>
        <item row="6" column="2">
         <widget class="QComboBox" name="comboBox_y"/>
        </item>
        <item row="6" column="1">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Y</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Left Bumper</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>B </string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Select Controller</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QComboBox" name="comboBox_x"/>
        </item>
        <item row="4" column="2">
         <widget class="QComboBox" name="comboBox_b"/>
        </item>
        <item row="19" column="2">
         <widget class="QComboBox" name="comboBox_dpright"/>
        </item>
        <item row="12" column="2">
         <widget class="QComboBox" name="comboBox_rstick"/>
        </item>
        <item row="2" column="2" colspan="2">
         <widget class="QComboBox" name="comboBox_Controllers"/>
        </item>
        <item row="9" column="2">
         <widget class="QComboBox" name="comboBox_rshoulder"/>
        </item>
        <item row="15" column="3">
         <widget class="QLabel" name="label_30">
          <property name="text">
           <string>Left Trigger</string>
          </property>
         </widget>
        </item>
        <item row="19" column="1">
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>DPad Right</string>
          </property>
         </widget>
        </item>
        <item row="14" column="1">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>DPad Up</string>
          </property>
         </widget>
        </item>
        <item row="11" column="3">
         <widget class="QLabel" name="label_26">
          <property name="text">
           <string>Right Analog Down</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>A</string>
          </property>
         </widget>
        </item>
        <item row="11" column="1">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Left Stick</string>
          </property>
         </widget>
        </item>
        <item row="15" column="5">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Triggers</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>X</string>
          </property>
         </widget>
        </item>
        <item row="15" column="1">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>DPad Down</string>
          </property>
         </widget>
        </item>
        <item row="12" column="1">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Right Stick</string>
          </property>
         </widget>
        </item>
        <item row="11" column="2">
         <widget class="QComboBox" name="comboBox_lstick"/>
        </item>
        <item row="3" column="2">
         <widget class="QComboBox" name="comboBox_a">
          <property name="minimumSize">
           <size>
            <width>177</width>
            <height>27</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="14" column="2">
         <widget class="QComboBox" name="comboBox_dpup"/>
        </item>
        <item row="8" column="3">
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string>Left Analog Right</string>
          </property>
         </widget>
        </item>
        <item row="15" column="2">
         <widget class="QComboBox" name="comboBox_dpdown"/>
        </item>
        <item row="9" column="4">
         <widget class="QComboBox" name="comboBox_rightyup"/>
        </item>
        <item row="14" column="4">
         <widget class="QComboBox" name="comboBox_rightxright"/>
        </item>
        <item row="8" column="4">
         <widget class="QComboBox" name="comboBox_leftxright"/>
        </item>
        <item row="11" column="4">
         <widget class="QComboBox" name="comboBox_rightydown"/>
        </item>
        <item row="14" column="3">
         <widget class="QLabel" name="label_28">
          <property name="text">
           <string>Right Analog Right</string>
          </property>
         </widget>
        </item>
        <item row="17" column="2">
         <widget class="QComboBox" name="comboBox_dpleft"/>
        </item>
        <item row="20" column="4">
         <widget class="QLabel" name="label_33">
          <property name="text">
           <string>File Name</string>
          </property>
         </widget>
        </item>
        <item row="6" column="3">
         <widget class="QLabel" name="label_22">
          <property name="text">
           <string>Left Analog Left</string>
          </property>
         </widget>
        </item>
        <item row="12" column="3">
         <widget class="QLabel" name="label_27">
          <property name="text">
           <string>Right Analog Left</string>
          </property>
         </widget>
        </item>
        <item row="23" column="2">
         <widget class="QComboBox" name="comboBox_guide"/>
        </item>
        <item row="4" column="3">
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>Left Analog Up</string>
          </property>
         </widget>
        </item>
        <item row="20" column="1">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item row="21" column="2">
         <widget class="QComboBox" name="comboBox_back"/>
        </item>
        <item row="4" column="4">
         <widget class="QComboBox" name="comboBox_leftyup"/>
        </item>
        <item row="9" column="3">
         <widget class="QLabel" name="label_25">
          <property name="text">
           <string>Right Analog Up</string>
          </property>
         </widget>
        </item>
        <item row="21" column="1">
         <widget class="QLabel" name="label_29">
          <property name="text">
           <string>Back</string>
          </property>
         </widget>
        </item>
        <item row="6" column="4">
         <widget class="QComboBox" name="comboBox_leftxleft"/>
        </item>
        <item row="5" column="4">
         <widget class="QComboBox" name="comboBox_leftydown"/>
        </item>
        <item row="12" column="5">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Right Analog</string>
          </property>
         </widget>
        </item>
        <item row="9" column="5">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Left Analog</string>
          </property>
         </widget>
        </item>
        <item row="3" column="5">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Mouse Sensitivity</string>
          </property>
         </widget>
        </item>
        <item row="3" column="6">
         <widget class="QDoubleSpinBox" name="doubleSpinBox_mouseSensitivity"/>
        </item>
        <item row="4" column="5">
         <widget class="QCheckBox" name="checkBox_invertmouse">
          <property name="text">
           <string>Invert Mouse</string>
          </property>
         </widget>
        </item>
        <item row="5" column="5">
         <widget class="QCheckBox" name="checkBox_dblscroll">
          <property name="text">
           <string>Double Scroll</string>
          </property>
         </widget>
        </item>
        <item row="8" column="5">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Dead Zones %</string>
          </property>
         </widget>
        </item>
        <item row="17" column="3">
         <widget class="QLabel" name="label_31">
          <property name="text">
           <string>Right Trigger</string>
          </property>
         </widget>
        </item>
        <item row="17" column="4">
         <widget class="QComboBox" name="comboBox_rtrigger"/>
        </item>
        <item row="15" column="4">
         <widget class="QComboBox" name="comboBox_ltrigger"/>
        </item>
        <item row="5" column="3">
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>Left Analog Down</string>
          </property>
         </widget>
        </item>
        <item row="12" column="4">
         <widget class="QComboBox" name="comboBox_rightxleft"/>
        </item>
        <item row="3" column="3">
         <widget class="QLabel" name="label_24">
          <property name="text">
           <string>MouseLook</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4" colspan="2">
         <widget class="QPushButton" name="pushButton_Refresh">
          <property name="text">
           <string>Refresh Controllers</string>
          </property>
         </widget>
        </item>
        <item row="23" column="6">
         <widget class="QPushButton" name="pushButton_delete">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QComboBox" name="comboBox_mouselook">
          <item>
           <property name="text">
            <string>Left Analog</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Right Analog</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="11" column="5">
         <widget class="QSpinBox" name="spinBox_leftAnalogDZ">
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="14" column="5">
         <widget class="QSpinBox" name="spinBox_rightAnalogDZ">
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="21" column="4" colspan="3">
         <widget class="QComboBox" name="comboBox_configfiles">
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="17" column="5">
         <widget class="QSpinBox" name="spinBox_triggerDZ">
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item row="23" column="5">
         <widget class="QPushButton" name="pushButton_save">
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item row="23" column="4">
         <widget class="QPushButton" name="pushButton_new">
          <property name="text">
           <string>New</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Debug</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QTextEdit" name="textEdit">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>815</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
